### ISSUE TITLE
Replace or augment KTooltip slot with a text prop

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri-design-system#221]
## Desired behavior

All examples I can find have plain text as the tooltip content. Therefore we should make a `text` prop the recommended way of passing in text:

```vue
<KTooltip
  reference="something"
  :refs="$refs"
  :text="someText"
/>
```

See also: https://github.com/learningequality/kolibri-design-system/issues/183

## Current behavior

Requires use of a verbose slot:

```vue
<KTooltip
  reference="something"
  :refs="$refs"
>
  {{ someText }}
</KTooltip>
```

## (Optional) The Value Add

Cleaner, more consistent API

**Comment by MisRob** [comment]
I'd suggest keeping the slot - if we ever need to pass HTML content via slots, it's good to have this option. But we can, for sure, add an option to pass text via property. Some other KDS components support both ways, so it's something that I would expect in regard to consistency.

**Comment by SandhyaR1007** [comment]
Hi is this issue still open? If yes I would like to work on this.

**Comment by MisRob** [comment]
Hello @SandhyaR1007, thank you for your interest in contributing. Yes, you're welcome to take it on. As soon as you start working on it, it'd be helpful if you posted a comment and I will assing you.

**Comment by FidalMathew** [comment]
@MisRob can I work on this issue?

**Comment by MisRob** [comment]
Hi @FidalMathew, thank you for volunteering! Yes, please add `text` property as a simpler alternative to the slot.

**Comment by FidalMathew** [comment]
Hi @MisRob and @LianaHarris360,
Can we close this issue? I think it's resolved.

**Comment by LianaHarris360** [comment]
Yes, we can close this issue now. Thank you for bringing it to our attention. Closed by #535
